scipy/stats/stats.py

```diff
@@ -4201,13 +4201,16 @@ def fisher_exact(table, alternative='two-sided'):
     --------
     chi2_contingency : Chi-square test of independence of variables in a
         contingency table.
+    contingency.odds_ratio : Compute the odds ratio (sample or conditional
+        MLE) for a 2x2 contingency table.
 
     Notes
     -----
-    The calculated odds ratio is different from the one R uses. This scipy
-    implementation returns the (more common) "unconditional Maximum
-    Likelihood Estimate", while R uses the "conditional Maximum Likelihood
-    Estimate".
+    The calculated odds ratio is different from the one R uses.  This
+    implementation returns the "unconditional maximum likelihood estimate",
```

Comment:
Is it ever referred to as the "sample" MLE? I only ask because the PR uses sample/conditional instead of "unconditional"/conditional elsewhere.